lib/rewardable/Reward.h

```diff
@@ -86,7 +86,7 @@ struct DLL_LINKAGE Reward
 	si32 calculateManaPoints(const CGHeroInstance * h) const;
 
 	Reward();
-	~Reward();
+	virtual ~Reward();
```

Comment:
Unnecessary? 
If this is to silence compiler warning then perhaps mark class as final?